test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampTests.cs

```diff
@@ -44,10 +44,12 @@ public async Task Timestamp_Verify_WithOfflineRevocation_ReturnsCorrectFlagsAndL
                 responders.Add(testServer.RegisterResponder(intermediateCa));
                 responders.Add(testServer.RegisterResponder(rootCa));
 
+                var storeLocation = CertificateStoreUtilities.GetTrustedCertificateStoreLocation();
```

Comment:
Use explicit type (`StoreLocation`) instead of `var`.

Reply:
Thanks! Fixed.